lib/features/auth/auth.dart

```diff
@@ -39,4 +39,6 @@ class Auth {
     _userData['user'] = response.data;
     return response.data;
   }
+
```

Comment:
logout method clears temp data from _userData variable not from cache
you have to clear all user data from cache